README.md

```diff
@@ -1 +1,41 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is the first project on the Microverse Learning path.
+
+![screenshot](./app_screenshot.png)
+
+This project contains an HTML page, styled with CSS
+and uses the Microverse linter config file.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] [OPTIONAL] It would also be nice to host your project on a live server like github pages so you can add a live project demo link